proto/types/dymint/state.proto

```diff
@@ -38,4 +38,17 @@ message State {
 
   uint64 last_store_height = 16 [(gogoproto.customname) = "LastStoreHeight"];
   uint64 base_height = 17;
+  RollappConsensusParams consensus_params = 18 [(gogoproto.nullable) = false];
+
+}
+
+message RollappConsensusParams {
+  Params params = 1;
```

Comment:
why do we need another `Params` message here vs just inlining the params ?

Reply:
it helps reading params from genesis state